Sources/Controller/Abstract/ModesBaseTableViewController.swift

```diff
@@ -37,21 +37,21 @@ class ModesBaseTableViewController: UITableViewController {
         title = Strings.Titles.Modes
     }
     
-    override func viewWillAppear(animated: Bool) {
-        self.clearsSelectionOnViewWillAppear = self.splitViewController!.collapsed
+    override func viewWillAppear(_ animated: Bool) {
+        self.clearsSelectionOnViewWillAppear = self.splitViewController!.isCollapsed
```

Comment:
Or maybe you wanted it to crash if it's not in a split view controller? (though I doubt it and optional or optional binding might be better for future)